frontend/src/components/maps/leaflet/ZoomOut/ZoomOutButton.tsx

```diff
@@ -28,9 +30,14 @@ export type ZoomOutProps = {
  * @param bounds The latlng bounds to zoom out to
  */
 export const ZoomOutButton: React.FC<ZoomOutProps> = ({ map, bounds }) => {
+  const dispatch = useAppDispatch();
+
   const zoomOut = () => {
     map.current?.leafletElement.fitBounds(bounds);
+    const zoom = map.current?.leafletElement.getZoom();
+    if (zoom) dispatch(setMapViewZoom(zoom));
```

Comment:
This is required or it will zoom back to the prior value after clicking zoom out.